loaders/html.py

```diff
@@ -0,0 +1,5 @@
+from .common import process_file
+from langchain.document_loaders import TextLoader
```

Comment:
why not using langchain URL loader, it would be better than loading html as text. HTML is usually pretty noisy

https://python.langchain.com/en/latest/modules/indexes/document_loaders/examples/url.html